src/lug/System/Win32/PathImplWin32.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <lug/System/Path.hpp>
+#include <windows.h>
+#include <Shlobj.h>
+
+namespace lug {
+namespace System {
+namespace priv {
+class PathImpl {

```

Comment:
Add newlines around the class definition, (to separate from the namespaces) and before each `public` or `private`